MUXSDKStats.json

```diff
@@ -1,3 +1,4 @@
 {
-	"2.11.0": "https://github.com/muxinc/mux-stats-sdk-avplayer/releases/download/v2.10.0/MUXSDKStats.xcframework.zip"
+	"2.11.0": "https://github.com/muxinc/mux-stats-sdk-avplayer/releases/download/v2.11.0/MUXSDKStats.xcframework.zip",
+    "2.12.0": "https://github.com/muxinc/mux-stats-sdk-avplayer/releases/download/v2.12.0/MUXSDKStats.xcframework.zip"
```

Comment:
formatting looks off?

Reply:
haha it does but I swear it looks fine on visual code and xcode so I'm not sure what's going on.
<img width="884" alt="Screen Shot 2022-03-07 at 14 51 45" src="https://user-images.githubusercontent.com/40036667/157116072-c0f95bd5-1cb2-4da9-9421-0be13ccaed42.png">
<img width="1006" alt="Screen Shot 2022-03-07 at 14 51 49" src="https://user-images.githubusercontent.com/40036667/157116074-8cb041d0-e15e-4922-92ae-9a795b3b6594.png">